src/components/mixins/console.ts

```diff
@@ -0,0 +1,62 @@
+import Vue from 'vue'
+import Component from 'vue-class-component'
+import { CommandHelp } from '@/store/printer/types'
+
+@Component
+export default class ConsoleMixin extends Vue {
+    get helplist(): CommandHelp[] {
+        return this.$store.state.printer.helplist ?? []
+    }
+
+    get consoleDirection() {
+        return this.$store.state.gui.console.direction ?? 'table'
+    }
+
+    get hideWaitTemperatures(): boolean {
+        return this.$store.state.gui.console.hideWaitTemperatures
+    }
+
+    set hideWaitTemperatures(newVal) {
+        this.$store.dispatch('gui/saveSetting', { name: 'console.hideWaitTemperatures', value: newVal })
+    }
+
+    get hideTlCommands(): boolean {
+        return this.$store.state.gui.console.hideWaitTemperatures
```

Comment:
guess this should be `hideTlCommands`
```suggestion
        return this.$store.state.gui.console.hideTlCommands
```